src/main/java/io/litmuschaos/response/UserWithProjectResponse.java

```diff
@@ -0,0 +1,90 @@
+package io.litmuschaos.response;
+
+import io.litmuschaos.response.ProjectResponse.CreatedBy;
+import io.litmuschaos.response.ProjectResponse.UpdatedBy;
+import java.util.List;
+
+public class UserWithProjectResponse {
+
+    private Long updatedAt;
+    private Long createdAt;
+    private CreatedBy createdBy;
+    private UpdatedBy updatedBy;
+    private boolean isRemoved;
+    private String id;
+    private String username;
+    private String email;
+    private String name;
+    private List<ProjectResponse> projects;
```

Comment:
@somln, please see `updatedAt` and `isRemoved` variable. you use Wrapper type for `updatedAt` and primitive type for `isRemoved`. I wondered why you use different type. Can you explain reason? 🙇‍♂️
(Wrapper type allow null value and primitive type didn't!)